doc/tutorials/starship.rst

```diff
@@ -5,12 +5,15 @@ Running Jobs with StarshipEngine
 
```

Comment:
Nope, there are all just standard ReST files. Just a matter of different contributors using different naming conventions. Since it doesn't show in the docs, I'm fine with leaving it as is.